src/colab/Run_SKAI_XManager_Colab_Pipeline.ipynb

```diff
@@ -103,50 +111,54 @@
       },
       "outputs": [],
       "source": [
-        "#@title Run XManager Train Job (with Vizier Hyperparameter Tuning) on Vertex AI\n",
+        "#@markdown Please make sure to chnage path of dataset\n",
         "\n",
         "#@markdown Enter arguments for the training job\n",
         "CONFIG_FILE = \"skai_two_tower_config\" #@param [\"skai_config\",\"skai_two_tower_config\"]\n",
         "DATASET_NAME = \"skai_dataset\" #@param {type:\"string\"}\n",
-        "DATASET_GCP_PATH = \"gs://skai-data/hurricane_ian\" #@param {type:\"string\"}\n",
-        "GCP_OUTPUT_DIR = \"gs://skai-data/experiments/skai_train_vizier\" #@param {type:\"string\"}\n",
+        "DATASET_LABELED_TRAIN = \"gs://bucket/path/to/examples/labeled_train_examples.tfrecord\" #@param {type:\"string\"}\n",
+        "DATASET_VALIDATION = \"gs://bucket/path/to/examples/labeled_train_examples.tfrecord\" #@param {type:\"string\"}\n",
+        "GCP_OUTPUT_DIR = \"gs://bucket/path/to/output/\" #@param {type:\"string\"}\n",
+        "GOOGLE_CLOUD_BUCKET_NAME = \"\" #@param {type:\"string\"}\n",
+        "EXPERIMENT_NAME = \"skai_train_xamanager\" #@param {type:\"string\"}\n",
         "NUM_EPOCHS = 10 #@param {type:\"integer\"}\n",
         "ACCELERATOR = \"V100\" #@param [\"V100\",\"T4\"]\n",
```

Comment:
If the use of TPU has not been tested, please remove this option for the moment.